setup.py

```diff
@@ -19,9 +21,21 @@
       url='https://github.com/AtteqCom/zsl',
       license='MIT',
       install_requires=[
-        'flask_injector==0.8.0',
-        'future'
+          'future',
+          'Flask-Injector>=0.8' if _is_py3 else 'Flask-Injector==0.8.0',
```

Comment:
Nice workaround, but it should be also done in requirements.txt so that Flask-Injector version would be the same with `pip install -r requirements.txt` and `pip install zsl`.